templates/blogs/supernav.html

```diff
@@ -2,6 +2,6 @@
     <p class="date-posted"><small>{{ entry.pub_date|date:"l, F j, Y" }}</small></p>
     <h4>{{ entry.title }}</h4>
     <p class="excerpt">
-        <small>{{ entry.summary|truncatewords_html:50|safe }}<a class="readmore" href="{{ entry.url }}">Read more</a></small>
+        <small>{{ entry.summary|truncatewords_html:50|striptags|safe }}<a class="readmore" href="{{ entry.url }}">Read more</a></small>
```

Comment:
There was a serious potential for xss here, locally I could see the images FeedBurner inserts to share a blogpost.